2021-12-05 16:09:27,385 - trainer.py[line:25] - INFO: Running with identifier 20211205_160927
2021-12-05 16:09:27,385 - trainer.py[line:27] - INFO: Saving log to log\rot360\20211205_160927\run.log
2021-12-05 16:09:27,385 - trainer.py[line:28] - INFO: Using args: [('log_name', 'rot360'), ('data_train', 'Shared/Rot360'), ('data_eval', None), ('output_checkpoint_dir', 'checkpoint'), ('eval_per_epoch', 20), ('log_dir', 'log'), ('tf_dir', 'tensorboard'), ('output_dir', 'output'), ('not_ignore_warning', False), ('enable_tensorboard', True), ('pack', True)]
2021-12-05 16:09:27,388 - trainer.py[line:39] - INFO: Using tensorboard, store at: tensorboard\rot360\20211205_160927
2021-12-05 16:09:27,388 - trainer.py[line:40] - INFO: Using training data: Shared/Rot360
2021-12-05 16:09:27,388 - trainer.py[line:42] - INFO: Pack this file using this command if training finished: --log_dir log --log_name rot360 --identifier 20211205_160927 --output_checkpoint_dir checkpoint --data_dir Shared/Rot360
2021-12-05 16:09:27,388 - train.py[line:97] - INFO: Running model resnet50
2021-12-05 16:09:27,389 - trainer.py[line:54] - ERROR: Cannot load file containing pickled data when allow_pickle=False
